### PR TITLE
Improve warning text about bad mbed htm

### PR DESCRIPTION
**Pull request by theotherjimmy**

### CODE REVIEW

File: mbed_lstools/lstools_base.py [hunk 1]
```diff
@@ -210,8 +210,7 @@ def _update_device_from_htm(self, device):
                             htm_target_id, device['target_id_usb_id'])
             device['target_id'] = htm_target_id
         else:
-            logging.warning("Could not read htm on from usb id %s. "
-                            "Falling back to usb id",
+            logging.warning("No HTM target ID found %s. Falling back to usb id",
```
Comment (bridadan) [inline mbed_lstools/lstools_base.py]: I thought we weren't using `logging` anymore, just the `logger` instance? Looks like `logging` is used a few times in this file.

Comment (theotherjimmy) [reply mbed_lstools/lstools_base.py]: Hmmmmmmmm. Don't know how that dropped out.

**Comment by theotherjimmy** [comment]
Looks like testing broke...

**Comment by theotherjimmy** [comment]
Nevermind, I had crap in my repo.

**Comment by theotherjimmy** [comment]
New warning:
```
# mbedls -s
WARNING:mbedls.lstools_base:No HTM target ID found. Falling back to usb id C006[Redacted]
 VBLUNO51  VBLUNO51[0]  /Yep/mount/point  /dev/ttyACM0  C006[Redacted]  0244 
```

**Comment by theotherjimmy** [comment]
With no MBED.HTM:
```
# mbedls -s
WARNING:mbedls.lstools_base:Failed to open file /yep/mount/point/mbed.htm
WARNING:mbedls.lstools_base:No HTM target ID found. Falling back to usb id C006[Redacted]
 VBLUNO51  VBLUNO51[0]  /yep/mount/point  /dev/ttyACM0  C006[Redacted]  0244 
```

**Comment by bridadan** [comment]
Can we a be a little more specific for the `No HTM target ID found` error? Specifically can we mention the path of the file we're looking for? For people new to mbed, they may not be familiar with the expected name of the HTM file.

**Comment by bridadan** [comment]
Let's try something along these lines:

```
WARNING:mbedls.lstools_base: No target ID found inside of the redirect URL in file <path to HTM file here>. Falling back to usb id C006[Redacted]
```

**Comment by theotherjimmy** [comment]
Yeah, sure. Just a moment.

**Comment by theotherjimmy** [comment]
That was a 32 min moment!

File: mbed_lstools/lstools_base.py [hunk 2]
```diff
@@ -206,13 +209,14 @@ def _update_device_from_htm(self, device):
...
                             htm_target_id, device['target_id_usb_id'])
             device['target_id'] = htm_target_id
         else:
-            logging.warning("Could not read htm on from usb id %s. "
-                            "Falling back to usb id",
-                            device['target_id_usb_id'])
+            logger.warning("No target ID found inside of the redirect URL in"
+                            " file %s. Falling back to usb id %s",
+                            device['target_id_usb_id'],
+                            join(mount_point, self.MBED_HTM_NAME))
```
Comment (bridadan) [inline mbed_lstools/lstools_base.py]: The supplied strings are in reversed order me thinks

Comment (theotherjimmy) [reply mbed_lstools/lstools_base.py]: Yep.

File: mbed_lstools/lstools_base.py [hunk 3]
```diff
@@ -206,12 +209,13 @@ def _update_device_from_htm(self, device):
...
-            logging.debug("Found htm target id, %s, for usb target id %s",
+            logger.debug("Found htm target id, %s, for usb target id %s",
                             htm_target_id, device['target_id_usb_id'])
             device['target_id'] = htm_target_id
         else:
-            logging.warning("Could not read htm on from usb id %s. "
-                            "Falling back to usb id",
+            logger.warning("No target ID found inside of the redirect URL in"
+                            " file %s. Falling back to usb id %s",
+                            join(mount_point, self.MBED_HTM_NAME),
```
Comment (bridadan) [inline mbed_lstools/lstools_base.py]: Jimmeh, `mount_point` doesn't exist here. I think you meant to index into `device`.

But the fact the test passed means that we don't have any coverage for this case either.

Comment (theotherjimmy) [reply mbed_lstools/lstools_base.py]: lol. no coverage!

Comment (theotherjimmy) [reply mbed_lstools/lstools_base.py]: I should really test these things.

Comment (theotherjimmy) [reply mbed_lstools/lstools_base.py]: I'll have to get back to this tomorrow. I have a website to deploy....

Comment (bridadan) [reply mbed_lstools/lstools_base.py]: No worries, I realize I'm adding more to your TODO list 😛 Thanks for keeping at it!

**Comment by theotherjimmy** [comment]
I'm going to close this PR as it's going to be superseded by work on #255

**Pull request closed without merging**